biojava-structure/src/main/java/org/biojava/nbio/structure/align/util/URLConnectionTools.java

```diff
@@ -119,72 +83,66 @@ public static HttpURLConnection openHttpURLConnection(URL url, int timeout)
 	 * @throws ConnectException
 	 *
 	 * */
-	public static HttpURLConnection openHttpURLConnection(URL url)
-	throws IOException, ConnectException {
+	public static URLConnection openURLConnection(URL url)
+			throws IOException, ConnectException {
```

Comment:
How about UrlConnectionTools?

On Thu, May 26, 2016 at 11:18 AM, Jose Manuel Duarte <
notifications@github.com> wrote:

> In
> biojava-structure/src/main/java/org/biojava/nbio/structure/align/util/URLConnectionTools.java
> <https://github.com/biojava/biojava/pull/494#discussion_r64794603>:
>
> > @@ -119,72 +83,66 @@ public static HttpURLConnection openHttpURLConnection(URL url, int timeout)
> >  	 * @throws ConnectException
> >  	 *
> >  	 * */
> > -	public static HttpURLConnection openHttpURLConnection(URL url)
> > -	throws IOException, ConnectException {
> > +	public static URLConnection openURLConnection(URL url)
> > +			throws IOException, ConnectException {
>
> ConnectException is not actually thrown by anything... it shouldn't be
> declared in the throws
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/biojava/biojava/pull/494/files/dde4915923602ed5e28e3a1db08b9cb584a4bea7#r64794603>
>



-- 
Peter Rose, Ph.D.
Site Head, RCSB Protein Data Bank West (http://www.rcsb.org)
San Diego Supercomputer Center (http://bioinformatics.sdsc.edu)
University of California, San Diego
+1-858-822-5497